projects/sandbox/pyproject.toml

```diff
@@ -162,6 +185,13 @@ device = "cuda"
 channel = "${base.channel}"
 basedir = "${base.basedir}"
 
-# arch parameters
-commands.coupling = "${base.coupling}"
-commands.maf = "${base.maf}"
+# flow subcommand
+[tool.typeo.scripts.sample-with-flow.flow]
+maf = "${base.maf}"
+coupling = "${base.coupling}"
+
+# embedding subcommand
+[tool.typeo.scripts.sample-with-flow.embedding]
+dense = "${base.dense}"
```

Comment:
I am a little confused here. Which one is being used? Asking since the `train:maf` under `steps` above does not specify the embedding.

Reply:
Yeah good question. So with this new `typeo` update, the `pinto.steps` table is deprecated. When alec writes a fix for that the syntax will look something like the following:

```toml
[[tool.pinto.steps]]
project = train
script = train
architecture = maf
embedding = resnet
optimizer = adam
scheduler = onecycler
 ...
```

So for now, the scripts need to be run by hand. The syntax for that is also slightly different:

```
pinto run train --typeo ../pyproject.toml script=train flow=maf embedding=resnet optimizer=adam scheduler=onecycle
```

Note that the order of the subcommands must match the order they are specified in the `@scriptify` wrapper